packages/adapter-neon/package.json

```diff
@@ -47,6 +47,6 @@
     "jest-junit": "16.0.0"
   },
   "peerDependencies": {
-    "@neondatabase/serverless": "^0.6.0 || ^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0"
+    "@neondatabase/serverless": ">0.6.0 <2"
```

Comment:
This reminds me: we have no policy about what versions of third-party drivers we want to support.